app/controllers/ItemController.scala

```diff
@@ -61,14 +61,17 @@ object ItemController extends Controller {
       throw new RuntimeException("You must define a Channel")
     }
 
+/*
```

Comment:
My preference is to just delete this type of stuff rather than comment it out. Version control and all... but either way works for me.